### ISSUE TITLE
linux build errors 

### ISSUE THREAD

**Issue by kmulvey** [CRVI/OpenCLIPP#9]
os: 3.18.7-100.fc20.x86_64

```
Building file: programs/ImageProximityFFT.cpp
g++ -g -O3 -Wall -c -fmessage-length=0 -std=c++0x -fPIC -I../include -I../include/c++ -I/usr/include -MMD -MP -MF"build/programs/ImageProximityFFT.deps" -MT"build/programs/ImageProximityFFT.deps" -o "build/programs/ImageProximityFFT.o" "programs/ImageProximityFFT.cpp"
programs/ImageProximityFFT.cpp:277:6: error: prototype for ‘void OpenCLIPP::ImageProximityFFT::MatchSquareDiff(int, int, OpenCLIPP::Image&, float, OpenCLIPP::Image&)’ does not match any in class ‘OpenCLIPP::ImageProximityFFT’
 void ImageProximityFFT::MatchSquareDiff(int , int , Image& , float , Image& )
      ^
In file included from programs/ImageProximityFFT.cpp:29:0:
../include/c++/Programs/ImageProximityFFT.h:95:9: error: candidate is: void OpenCLIPP::ImageProximityFFT::MatchSquareDiff(int, int, OpenCLIPP::Image&, double*, OpenCLIPP::Image&)
    void MatchSquareDiff(int width, int hight, Image& Source, double * templ_sqsum, Image& Dest);
         ^
programs/ImageProximityFFT.cpp:280:6: error: prototype for ‘void OpenCLIPP::ImageProximityFFT::MatchSquareDiffNorm(int, int, OpenCLIPP::Image&, float, OpenCLIPP::Image&)’ does not match any in class ‘OpenCLIPP::ImageProximityFFT’
 void ImageProximityFFT::MatchSquareDiffNorm(int , int , Image& , float , Image& )
      ^
In file included from programs/ImageProximityFFT.cpp:29:0:
../include/c++/Programs/ImageProximityFFT.h:98:9: error: candidate is: void OpenCLIPP::ImageProximityFFT::MatchSquareDiffNorm(int, int, OpenCLIPP::Image&, double*, OpenCLIPP::Image&)
    void MatchSquareDiffNorm(int width, int hight, Image& Source, double * templ_sqsum, Image& Dest);
         ^
programs/ImageProximityFFT.cpp:283:6: error: prototype for ‘void OpenCLIPP::ImageProximityFFT::MatchCrossCorrNorm(int, int, OpenCLIPP::Image&, float, OpenCLIPP::Image&)’ does not match any in class ‘OpenCLIPP::ImageProximityFFT’
 void ImageProximityFFT::MatchCrossCorrNorm(int , int , Image& , float , Image& )
      ^
In file included from programs/ImageProximityFFT.cpp:29:0:
../include/c++/Programs/ImageProximityFFT.h:101:9: error: candidate is: void OpenCLIPP::ImageProximityFFT::MatchCrossCorrNorm(int, int, OpenCLIPP::Image&, double*, OpenCLIPP::Image&)
    void MatchCrossCorrNorm(int width, int hight, Image& Source, double * templ_sqsum, Image& Dest);
         ^
make[1]: *** [build/programs/ImageProximityFFT.o] Error 1
make[1]: Leaving directory `/opt/code/OpenCLIPP/C++'
make: *** [C++] Error 2

```

**Comment by kmulvey** [comment]
that was from using the code from:
https://github.com/CRVI/OpenCLIPP/releases/tag/v2.0.0

**Comment by AntAgna** [comment]
Thank you for reporting.
I believe this has been fixed this commit : 6350180ca7aeb1a86a9ce3a2dd1b6ca87e0dd244
